src/app/about/page.js

```diff
@@ -27,7 +27,7 @@ const About = () => {
       {sections.map((section, index) => (
         <div
           key={index}
-          className="flex flex-col mx-auto w-1/2 text-center pb-12"
+          className="flex flex-col mx-auto md:leading-loose w-1/2 md:w-4/5 md:text-xs text-center pb-12"
```

Comment:
break points apply to larger screen sizes rather than smaller screen sizes.

so leading loose shouldn't have a breakpoint, but there should be a break point for regular leadings for larger screen sizes.